src/Integration.UnitTests/VsSessionHostTests.cs

```diff
@@ -136,6 +135,47 @@ public void VsSessionHost_ClearActiveSection_ClearState()
         }
 
         [TestMethod]
+        public void VsSessionHost_ActiveSectionChangedEvent()
```

Comment:
I need the event in order to start the binding process when the section exists (and not busy connecting)